test/integration/patients/worklist/worklist.js

```diff
@@ -2555,15 +2555,15 @@ context('worklist page', function() {
           type: 'patients',
           attributes: {
             first_name: 'Patient',
-            last_name: 'Field 1',
+            last_name: 'Field A',
           },
           relationships: { 'patient-fields': { data: [{ id: '1' }] } },
         });
 
         fx.included.push({
           id: '1',
           type: 'patient-fields',
-          attributes: { value: { value: 1 }, name: 'foo' },
+          attributes: { value: { value: 'A' }, name: 'foo' },
```

Comment:
These two changes shouldn't have been reverted.

Reply:
Because of time constraints, just reverted them back so they'll pass. I'll create a separate story to come back and update it.